apps/prairielearn/src/tests/sync/courseInstancesSync.test.ts

```diff
@@ -132,6 +132,136 @@ describe('Course instance syncing', () => {
     assert.equal(remainingRule.institution, 'Any');
   });
 
+  describe('syncs access control settings correctly', async () => {
```

Comment:
Nit: we should be consistent with naming, IMO we should use "publishing settings" or "publishing config" instead of "access control settings".